### PR TITLE
[OSDOCS-11982]AWS c7a,r7a,hpc,c7i,r7i,g4ad as Worker Node Instance types on ROSA/OSD

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.16+
Issue:

https://issues.redhat.com/browse/OSDOCS-11982
Link to docs preview:

https://81752--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/osd-service-definition.html#aws-instance-types-ccs_osd-service-definition (OSD ON AWS)

https://81752--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/osd-service-definition.html#aws-instance-types-ccs_osd-service-definition (ROSA) 
QE review: 
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by arendej — APPROVED** [review]
lgtm

**Comment by AkashKanni** [comment]
lgtm

File: modules/rosa-sdpolicy-am-aws-compute-types.adoc [hunk 1]
```diff
@@ -376,6 +400,33 @@ Support for the GPU instance type software stack is provided by AWS. Ensure that
 - m5zn.3xlarge (16 vCPU, 48 GiB)
 - m5zn.6xlarge (32 vCPU, 96 GiB)
 - m5zn.xlarge (4 vCPU, 16 GiB)
+- c7a.xlarge (4 vCPU, 8 GiB)
```
Comment (bergerhoffer) [inline modules/rosa-sdpolicy-am-aws-compute-types.adoc:403]: Not sure if it's supposed to be alphabetical order or not - just checking if this c7a section should go after the c6 section (and before the m5 section)?

**Review by bergerhoffer — COMMENTED** [review]
Just one question!

File: modules/rosa-sdpolicy-am-aws-compute-types.adoc (hunk 1 above)
Comment (mletalie) [reply modules/rosa-sdpolicy-am-aws-compute-types.adoc:398]: Hi @bergerhoffer,
     Not necessarily. There is not a hard set rule for additional instance types. However, I did re-arrange that  particular category.

**Review by Srivaralakshmi — COMMENTED** [review]
/lgtm

**Pull request merged**